src/components/MovieCard/moviecard.js

```diff
@@ -34,10 +36,12 @@ const MovieCard = ({id, title, description, year, image, releaseDate, runtime, g
     { title: 'Delete', callback: toggleDelete },
   ];
   
+  
   return (
     <div className="movie-card">
       <div className="image-container">
-        <img onClick={onClick} className="movie-image" src={image} alt={title} />
+        <img onClick={function(event){ onClick(); history.push(`/movie/${id}`); }}
```

Comment:
I'd suggest to extract toggleMovieHeader to redux as well, so you can make it a https://reactrouter.com/web/api/Link

Reply:
done